db/migrate/20250715193037_add_device_resolution_lookup_table.rb

```diff
@@ -0,0 +1,34 @@
+class AddDeviceResolutionLookupTable < ActiveRecord::Migration[7.1]
+  def up
+    create_table "land.device_resolutions", id: :uuid, default: 'gen_random_uuid()', primary_key: :device_resolution_id do |t|
+      t.string :device_resolution, index: true 
+      t.integer :width, index: true
+      t.integer :height, index: true
+      t.string :orientation, index: true
+
+      t.timestamps
+    end
+
+    add_index "land.device_resolutions", [:width, :height, :orientation], name: 'index_device_resolutions_on_dims_and_orientation'
+
+    add_column "land.user_agents", :device_resolution_id, :uuid
+    add_index "land.user_agents", :device_resolution_id, name: 'index_user_agents_on_device_resolution_id'
+
+    add_column "land.browsers", :dark_mode, :boolean
+    add_column "land.browsers", :light_mode, :boolean
+    add_column "land.browsers", :no_preference, :boolean
+    add_index "land.browsers", [:dark_mode, :light_mode, :no_preference], name: 'index_browsers_on_color_scheme_preferences'
+  end
+
+  def down
+    remove_index "land.browsers", name: 'index_browsers_on_color_scheme_preferences'
+    remove_column "land.browsers", :no_preference
+    remove_column "land.browsers", :light_mode
+    remove_column "land.browsers", :dark_mode
+
+    remove_index "land.user_agents", name: 'index_user_agents_on_device_resolution_id'
+    remove_column "land.user_agents", :device_resolution_id
+
+    drop_table "land.device_resolutions", if_exists: true
+  end
```

Comment:
This can be moved to use `#change` as all methods are active record methods (I believe):
```suggestion
  def change
    create_table "land.device_resolutions", id: :uuid, default: 'gen_random_uuid()', primary_key: :device_resolution_id do |t|
      t.string :device_resolution, index: true 
      t.integer :width, index: true
      t.integer :height, index: true
      t.string :orientation, index: true

      t.timestamps
    end

    add_index "land.device_resolutions", [:width, :height, :orientation], name: 'index_device_resolutions_on_dims_and_orientation'

    add_column "land.user_agents", :device_resolution_id, :uuid
    add_index "land.user_agents", :device_resolution_id, name: 'index_user_agents_on_device_resolution_id'

    add_column "land.browsers", :dark_mode, :boolean
    add_column "land.browsers", :light_mode, :boolean
    add_column "land.browsers", :no_preference, :boolean
    add_index "land.browsers", [:dark_mode, :light_mode, :no_preference], name: 'index_browsers_on_color_scheme_preferences'
  end
```

Reply:
Yes! `create_table`, `add_index`, and `add_column` are all ActiveRecord methods so I will update this!